nms/app/views/login/LoginForm.tsx

```diff
@@ -65,67 +102,117 @@ class LoginForm extends React.Component<Props> {
     const params = new URLSearchParams(window.location.search);
     const to = params.get('to');
 
-    if (ssoEnabled) {
-      return (
-        <Card raised={true} className={classes.card}>
-          <CardContent>
-            <Text className={classes.title} variant="h5">
-              {this.props.title}
-            </Text>
-            {error}
-          </CardContent>
-          <CardActions className={classes.footer}>
-            <Button
-              variant="contained"
-              color="primary"
-              onClick={() => {
-                window.location.href =
-                  (ssoAction || '') + window.location.search;
-              }}>
-              Sign In
-            </Button>
-          </CardActions>
-        </Card>
-      );
-    }
+    const [organization] = window.location.hostname.split('.', 1);
+    const hostPortal: boolean = organization === 'host';
 
     return (
-      <Card raised={true} className={classes.card}>
-        <form
-          ref={ref => (this.form = ref)}
-          method="post"
-          action={this.props.action}>
-          <input type="hidden" name="_csrf" value={csrfToken} />
-          <input type="hidden" name="to" value={to!} />
-          <CardContent>
-            <Text className={classes.title} variant="h5">
-              {this.props.title}
-            </Text>
-            {error}
-            <TextField
-              name="email"
-              label="Email"
-              className={classes.input}
-              onKeyUp={key => key.keyCode === ENTER_KEY && this.form?.submit()}
-            />
-            <TextField
-              name="password"
-              label="Password"
-              type="password"
-              className={classes.input}
-              onKeyUp={key => key.keyCode === ENTER_KEY && this.form?.submit()}
-            />
-          </CardContent>
-          <CardActions className={classes.footer}>
-            <Button
-              onClick={() => this.form?.submit()}
-              variant="contained"
-              color="primary">
-              Login
-            </Button>
-          </CardActions>
-        </form>
-      </Card>
+      <div className={classes.login}>
+        {ssoEnabled ? (
+          <>
+            <div className={classes.title}>
```

Comment:
Ins't the title/sub-title exactly the same as without SSO and can therefore be moved out of ternary?